README.md

```diff
@@ -1,127 +1,194 @@
-Returns transaction, which is a Nethereum DataContract containing all the needed information about transaction. 
+The SDK is designed to make it super easy to get started with Game development by enabling connection and interaction across different blockchains. 
 
-```c#
-Transaction trx = await contract.GetTransactionInfo(receipt);
-```
----
-* Task<TReturnType> GetData<FieldData, ReturnType>(FieldData requestData = null)*<br>
-To get data from mappings and call contract methods that no need of mining use this method.
+ * Contains a huge range of classes, plugins and example scripts for a variety of use cases. 
 
-You need to prepare DTO class based on the arguments of a contract method.
+* Nethereum libraries provide support for web requests using RPC over Http. 
 
-Contract method:
-```sol
-function balanceOf(address owner) public view virtual override returns (uint256)
-```
+* Ankr RPC network infrastructure provides fast and easy connection to multiple chains. 
+
+
+## Getting started
+
+### 🧰 Prerequisites
+
+1. Smart Contracts must already be deployed on the blockchain. You have the Smart Contract addresses and ABI
+
+###  📌 Use Cases
+
+This help content focuses on the following use cases. 
+
+1. Connecting to a Web3 Wallet (MetaMask) and Authenticating a user
+
+2. Performing Updates to NFTs by interacting with the blockchain and calling smart functions.
+
+
+<!-- 
+1. Examples Scripts, 
+*  The *Scripts* folder contains a wide range of useful scripts for initializing sessions, creating and managing Game NFTs and Scenes and authenticating sessions via a Web3 Wallet e.g. Metamask. 
+`ContractMessage` scripts for retrieving information for and managing NFTs  e.g. Game Characters on the blockchain.  ERC-721 and ERC-1155 
+`DTO` scripts for handling TransferObject events and accessing logs.
+`ERC-20`folder contains example script for minting new NFTs e.g. characters
+`ERC-721` folder contains example scripts for managing ownership of NFTs.
+`WearableNFTExample` folder contains example scripts for minting and managing NFT character apparel e.g. hats. 
+`ConnectionController` and `CustomSceneManager` ensure ongoing authentication via MetaMask and displaying game scenes. 
+to allow smart contract interfacing with the for building NFTs to the Blockchain.  scripts to ERC721 and ERC1155 scriptsScripts that can be used  -->
+
+## 👝 01 Connect Web3 Wallet
+
+Connecting to an Ethereum i.e. MetaMask wallet provides a link between a wallet address and a user's game account.
+
+1. Create an instance of a `AnkrSDKWrapper` class via `AnkrSDKWrapper.GetSDKInstance(...)` method after successful login in metamask
 
-DTO:
 ```c#
-[Function("balanceOf", "uint256")]
-public class BalanceOfMessage : FunctionMessage
-{
-	[Parameter("address", "_owner", 1)]
-	public virtual string Owner { get; set; }
-}
+string provider_url = "<ethereum node url>";
+		
+Web3 web3 = new Web3(provider_url);
+web3.Initialize();
```

Comment:
should be changed with 
var mirageSDKWrapper = MirageSDKWrapper.GetSDKInstance(Contract.ProviderURL);

Reply:
ankr*

Reply:
yeah still need to change that class's name too